src/components/common/match-item.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../style/common/constants';
```

Comment:
don't you use absolute paths?

Reply:
Yes but in js files. In scss files I couldn't change that